app/templates/views/dashboard/dashboard.html

```diff
@@ -22,16 +22,29 @@ <h2 class="font-body-2xl line-height-sans-2 margin-0">
       Messages sent
     </h2>
 
-    <!-- <button id="sevenDaysButton">7 Days</button>
-    <canvas id="myChart"></canvas> -->
     {{ ajax_block(partials, updates_url, 'inbox') }}
 
     {{ ajax_block(partials, updates_url, 'totals') }}
 
+    <div id="chartContainer" data-sms-sent="{{ sms_sent }}" data-sms-allowance-remaining="{{ sms_allowance_remaining }}">
```

Comment:
We don't need these piece of code since we aren't passing it this way anymore 

`data-sms-sent="{{ sms_sent }}" data-sms-allowance-remaining="{{ sms_allowance_remaining }}"`